src/chocolatey.resources/helpers/functions/Get-Installed.ps1

```diff
@@ -0,0 +1,46 @@
+﻿## Get-Installed
+##############################################################################################################
+## Returns a list of installed applications.
+## Registry based replacement for "Get-WmiObject -Class Win32_Product".
+##############################################################################################################
+
+function Get-Installed {
+param(
+  [string] $NonMSIOption = 'HideNonMSI' # You may use 'ShowNonMSI' option

```

Comment:
Wrapped Get-WMIObject behaves similarly. Get-Installed is another thing and it have the option.